UnitTests/Tests/WeatherUpdateCacheSpec.swift

```diff
@@ -0,0 +1,68 @@
+@testable import Tropos
+import Quick
+import Nimble
+
+private let testCacheFileName = "TestWeatherUpdate"
+
+private var testCachesDirectory: NSURL {
+    return NSFileManager.defaultManager()
+        .URLsForDirectory(.DocumentDirectory, inDomains: .UserDomainMask)
+        .first!
+}
+
+private var testWeatherUpdateURL: NSURL {
+    return testCachesDirectory.URLByAppendingPathComponent(testCacheFileName)
+}
+
+private func resetFilesystem() {
+    do {
```

Comment:
Have you considered replacing this with a `try!`. This is cleaning up state, I would want the current test to fail if for some reason the cleanup fails. 

Swallowing the error could mean a later test fails because an earlier test didn't correctly clean up the state.